adobe/cmath.hpp

```diff
@@ -5,170 +5,33 @@
 */
 /*************************************************************************************************/
 
-/*
-
-REVISIT (sparent) : Need to replicate the boost configuration tests to figure out when to fall
-back to include math.h. This also needs to add any other C99 math.h extensions.
-
-*/
-
 #ifndef ADOBE_CMATH_HPP
 #define ADOBE_CMATH_HPP
 
 #include <adobe/config.hpp>
 
+#include <cmath>
 #include <functional>
 
 /*************************************************************************************************/
 
-#if defined(__MWERKS__)
-/*
-    Any (previously) supported version of metrowerks had the C99/TR1 cmath extensions in the
-    standard namespace in <cmath>.
-*/
 #define ADOBE_HAS_C99_STD_MATH_H
```

Comment:
I check this tonight.
GIven that only VS2015 update 2 can handle ASL (with patches, as far as I remember), can I only test with this one ?
By the way, for GCC6 support I could just edit the "_GNUC_ == 5" to >= 5.
I cleaned here because Clang builds under Linux are broken with current version checks. Clang seems to report a low _GNUC_ version, and yet has latest cmath support.